tcms/tests/__init__.py

```diff
@@ -227,3 +230,47 @@ def setUpTestData(cls):
         cls.execution_4 = executions[3]
         cls.execution_5 = executions[4]
         cls.execution_6 = executions[5]
+
+
+class GetListOffAllPossiblePermissions(BaseCommand):
+
+    # Helper class for getting all possible permissions
+    # Taken from:
+    # timonweb.com/posts/how-to-get-a-list-of-all-user-permissions-available-in-django-based-project/
+
+    def handle(self, *args, **options):
+        permissions = set()
+
+        tmp_superuser = get_user_model()(
+            is_active=True,
+            is_superuser=True
+        )
+
+        for backend in auth.get_backends():
+            if hasattr(backend, "get_all_permissions"):
+                permissions.update(backend.get_all_permissions(tmp_superuser))
+
+        sorted_list_of_permissions = sorted(list(permissions))
+
+        return sorted_list_of_permissions
+
+
+class CreateTestUsers(LoggedInTestCase):
+    """Base test class for user creation"""
+
+    @classmethod
+    def setUpTestData(cls):
+        super(CreateTestUsers, cls).setUpTestData()
```

Comment:
Use the Python 3 form of super which is `super().<method>`

Reply:
Also calling super without doing anything else here is not necessary. The point of overriding methods is to be able to change some of the behavior.